plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -443,9 +459,10 @@ func TestRemovePod(t *testing.T) {
 
 func BenchmarkGetNodeNameToInfoMap1kNodes30kPods(b *testing.B) {
 	cache := setupCacheOf1kNodes30kPods(b)
+	nodeInfoMap := make(map[string]*NodeInfo)
 	b.ResetTimer()
 	for n := 0; n < b.N; n++ {
-		cache.GetNodeNameToInfoMap()
+		cache.UpdateNodeNameToInfoMap(nodeInfoMap)
```

Comment:
That is what I feel. The gain is like 100% in the bench. So shall we just remove this benchmark?